front/components/LP/top.tsx

```diff
@@ -11,7 +11,7 @@ export default class App extends React.Component<AppProps, any> {
       <div className="top">
         <h1 className="title">SD Scheduler</h1>
         <div className="description">
-          Schedule final presentation without&nbsp;stress.
+          Final presentation scheduling without&nbsp;the&nbsp;stress
```

Comment:
I looked up the bunch of the company website and the catch copy starts with the sentence. I think that makes the sentence more powerful.